x-pack/plugin/sql/jdbc/src/main/java/org/elasticsearch/xpack/sql/jdbc/JdbcResultSet.java

```diff
@@ -547,7 +550,9 @@ public InputStream getBinaryStream(int columnIndex) throws SQLException {
     @Override
     @Deprecated
     public BigDecimal getBigDecimal(String columnLabel, int scale) throws SQLException {
-        throw new SQLFeatureNotSupportedException("BigDecimal not supported");
+        BigDecimal bd = getBigDecimal(columnLabel);
```

Comment:
Why not delegate to the other `getBigDecimal` -> `return getBigDecimal(column(columnLabel), scale)` ?

Reply:
Right, good point! Thanks.